test/uk/gov/hmrc/brm/controllers/BirthEventsControllerSpec.scala

```diff
@@ -524,30 +525,32 @@ class BirthEventsControllerSpec
           bodyOf(result) shouldBe empty
         }
 
-        "return 200 match false when GRO returns Forbidden 418 Teapot body" in {
+        "return 503 when GRO throws UpstreamInternalServerError" in {
           when(mockAuditConnector.sendEvent(Matchers.any())(Matchers.any(), Matchers.any())).thenReturn(Future.successful(AuditResult.Success))
           when(MockController.service.groConnector.getReference(Matchers.any())(Matchers.any()))
-            .thenReturn(Future.failed(Upstream4xxResponse(MockErrorResponses.TEAPOT.json, FORBIDDEN, FORBIDDEN)))
+            .thenReturn(Future.failed(new Upstream5xxResponse(MockErrorResponses.CONNECTION_DOWN.json, INTERNAL_SERVER_ERROR, INTERNAL_SERVER_ERROR)))
```

Comment:
@manishsw I would like to talk about this one as a team as it could be common to get a 500 response from GRO. Think this is correct just want to discuss